docs/FAQ.md

```diff
@@ -43,10 +43,10 @@ Etcher requires an available [polkit authentication agent](https://wiki.archlinu
 
 ## May I run Etcher in older macOS versions?
 
-Etcher GUI is based on the [Electron](http://electron.atom.io/) framework, [which only supports macOS 10.10 and newer versions](https://github.com/electron/electron/blob/master/docs/tutorial/support.md#supported-platforms).
+Etcher GUI is based on the [Electron](https://www.electronjs.org/) framework, [which only supports macOS 10.10 and newer versions](https://github.com/electron/electron/blob/main/README.md#platform-support).
```

Comment:
That link says "macOS (Catalina and up)" and https://en.wikipedia.org/wiki/MacOS_Catalina says that this is macOS 10.15 ?